.eslintrc

```diff
@@ -1,36 +1,36 @@
-{
-  "extends": "mourner",
-  "parser": "babel-eslint",
-  "parserOptions": {
-    "ecmaVersion": 6,
-    "sourceType": "module"
-  },
-  "globals": {
-    "document": true
-  },
-  "rules": {
-    "indent": [2,2],
-    "array-bracket-spacing": "off",
-    "block-scoped-var": "error",
-    "consistent-return": "off",
-    "global-require": "off",
-    "key-spacing": "off",
-    "no-eq-null": "off",
-    "no-new": "off",
-    "no-var": "error",
-    "no-warning-comments": "error",
-    "object-curly-spacing": "off",
-    "prefer-arrow-callback": "error",
-    "prefer-const": "error",
-    "prefer-template": "error",
-    "quotes": "off",
-    "space-before-function-paren": "off",
-    "template-curly-spacing": "error",
-    "camelcase": 0,
-    "no-console": ["error", { "allow": ["warn", "error"] }]
-  },
-  "env": {
-    "es6": true,
-    "browser": true
-  }
-}
+// {
```

Comment:
This was causing me bugs with npm link. idk if people think I should uncomment it or delete it?

Reply:
@msimms1123 When you say:
> This was causing me bugs with npm link
Do you mean that when `npm-link`ing this repo to Vetro 2 front end, vetro 2 front end ESLint was complaining about lint issues in mapbox-gl-ddaw?

If yes, I think I know how to solve the issue. (I’m at the allergist though, and so have not tried this).
- vetro 2 ESLint is complaining because the package is not actually in your node modules (the sym link is in node modules, but the real package is not) See https://stackoverflow.com/a/60111971/2747370
- I’m not sure how to make the answer in that SO post work. But one way that I think would work is to tell vetro 2 ESLint to ignore mapbox-gl-draw. Two ways you could do that:

    1. Add a `.eslintignore` to vetro 2, and add something like `**/mapbox-gl-draw` to it. I’m not sure the exact glob pattern
    2. Or edit our `.eslintrc` and use `ignorePatterns` at the top level with a similar glob, see https://eslint.org/docs/user-guide/configuring#ignoring-files-and-directories

Reply:
(And if this works, it should let you keep the existing map box-gl-draw ESLint. Which seems better to me. Seems like we should try to mess with their repo as little as possible, to make merging future releases into ours easier.)